test/compilable/cppmangle.d

```diff
@@ -995,3 +995,18 @@ version (Win64)
 {
     static assert(test_char_mangling.mangleof == "?test_char_mangling@@YAXD_S_U_W@Z");
 }
+
+/**************************************/
+// https://issues.dlang.org/show_bug.cgi?id=15505
+// extern(C++) array parameter mangling gains surprise const
+
+extern(C++) void issue15505(int[4]* t);
+
+version (Win64)
```

Comment:
Isn't it better to use `CRuntime_Microsoft`? The bug says it only occurs when targeting COFF.

Reply:
No because the mangling is different between 32 and 64bit. The (correct) mangling also the same as with DMC (I think).

Reply:
These tests (checking against hardcoded mangled name **assumed** to be correct) are awful and haven't worked out before. A real linker test is robust and doesn't need any versioning.

Reply:
I did cross-reference them on godbolt.